src/jsaf/util/Regex.java

```diff
@@ -25,319 +23,634 @@
  * @since 1.3.8
  */
 public class Regex {
-    /**
-     * @since 1.6.10
-     */
-    public static final Pattern WILDCARD = Pattern.compile(".*");
 
-    protected static final char cOBRACE = '{';
-    protected static final char cCBRACE = '}';
-    protected static final char cOPEREN = '(';
-    protected static final char cCPEREN = ')';
-    protected static final char cOBRACK = '[';
-    protected static final char cCBRACK = ']';
+  protected static final String OBRACE = "{";
+  protected static final String CBRACE = "}";
+  private static final String ESCAPE = "\\";
+  private static final String QUALIFIER_PATTERN = "[0-9]+,{0,1}[0-9]*";
 
-    protected static final String szOBRACE = "{";
-    protected static final String szCBRACE = "}";
-    protected static final String szOPEREN = "(";
-    protected static final String szCPEREN = ")";
-    protected static final String szOBRACK = "[";
-    protected static final String szCBRACK = "]";
+  // Characters that have special meanings within a regex for which escaping is required to match the literal character.
+  private static final String[] SPECIAL_CHARACTERS = { ESCAPE, "^", ".", "$", "|", "(", ")", "[", "]", OBRACE, CBRACE, "*", "+", "?" };
 
-    private static final String ESCAPE = "\\";
-    private static final String QUALIFIER_PATTERN = "[0-9]+,{0,1}[0-9]*";
+  public static enum REGEX_CONCEPT {
```

Comment:
It just occurred to me that having an ALL-CAPS name for the enum breaks the convention in-use in the codebase to use CamelCase for enum names.  We use ALL CAPS only for enum instances and static final variables.

Nevertheless we're under the gun to get a release out today, so I'll let you decide what to do.